packages/contracts/contracts/test/MajorityVotingMock.sol

```diff
@@ -5,38 +5,26 @@ pragma solidity 0.8.10;
 import "../voting/majority/MajorityVotingBase.sol";
 
 contract MajorityVotingMock is MajorityVotingBase {
-    function initializeMock(
-        IDAO _dao,
-        uint64 _supportThreshold,
-        uint64 _minParticipation,
-        uint64 _minDuration,
-        uint256 _minProposerVotingPower
-    ) public initializer {
-        __MajorityVotingBase_init(
-            _dao,
-            _supportThreshold,
-            _minParticipation,
-            _minDuration,
-            _minProposerVotingPower
-        );
+    function initializeMock(IDAO _dao, PluginSettings calldata _pluginSettings) public initializer {
```

Comment:
`PluginSettings` feels too generic for a name. Then should be plugin-independent, shouldn't they?

Reply:
Yes, I find this not so fitting as well which is why I opened this thread: 
https://discord.com/channels/672466989217873929/950422827893420153/1052283843031339038

Reply:
mmm ok, but the renaming PR is one thing... yet this is new code that we are introducing, so I would pick a proper name here. 

IMO it should be `MajorityVotingMockPluginSettings`, then `TokenVotingPluginSettings`, ... `_____PluginSettings` with whichever specific settings that plugin needs.

Reply:
As `TokenVoting` and `AddresslistVoting` both use these settings, the proper name would be `MajorityVotingSettings` then.

Reply:
Renamed to `VotingSettings` in c69aa38770f33a15baed838deca116c0c78f7e22